proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -1,21 +1,85 @@
 {
   "ams": {
     "color": {
-      "primary-black": { "value": "#000000" },
-      "primary-white": { "value": "#FFFFFF" },
-      "primary-blue": { "value": "#004699" },
-      "primary-red": { "value": "#EC0000" },
-      "dark-blue": { "value": "#102E62" },
-      "orange": { "value": "#FF9100" },
-      "yellow": { "value": "#FFE600" },
-      "green": { "value": "#BED200" },
-      "dark-green": { "value": "#00A03C" },
-      "blue": { "value": "#009DE6" },
-      "purple": { "value": "#A00078" },
-      "magenta": { "value": "#E50082" },
-      "neutral-grey1": { "value": "#E8E8E8" },
-      "neutral-grey2": { "value": "#BEBEBE" },
-      "neutral-grey3": { "value": "#767676" }
+      "border": {
+        "default": { "value": "#181818" }
+      },
+      "canvas": {
+        "default": { "value": "#ffffff" }
+      },
+      "feedback": {
+        "error": {
+          "default": { "value": "#ec0000" }
+        },
+        "info": {
+          "default": { "value": "#009de6" }
+        },
+        "success": {
+          "default": { "value": "#00a03c" }
+        },
+        "warning": {
+          "default": { "value": "#ff9100" }
+        }
+      },
+      "footer": {
+        "default": { "value": "#004699" }
+      },
+      "highlight": {
+        "azure": {
+          "default": { "value": "#009de6" }
+        },
+        "green": {
+          "default": { "value": "#00a03c" }
+        },
+        "lime": {
+          "default": { "value": "#bed200" }
+        },
+        "magenta": {
+          "default": { "value": "#e50082" }
+        },
+        "orange": {
+          "default": { "value": "#ff9100" }
+        },
+        "purple": {
+          "default": { "value": "#a00078" }
+        },
+        "yellow": {
+          "default": { "value": "#ffe600" }
+        }
+      },
+      "interactive": {
+        "default": { "value": "#004699" },
+        "hover": { "value": "#003677" },
+        "contrast": {
+          "default": { "value": "#000000" }
+        },
+        "disabled": {
+          "default": { "value": "#767676" }
+        },
+        "invalid": {
+          "default": { "value": "#ec0000" },
+          "hover": { "value": "#b70000" }
+        },
+        "inverse": {
+          "default": { "value": "#ffffff" }
+        }
+      },
+      "logo": {
```

Comment:
In some cases there is black text in the logo (satellites). Do we need that color from logo or just get the default text color?

Reply:
Yes, default text colour. These are not component tokens.

Reply:
Then the fact that you have a group here called "logo" is confusing (me). Maybe something like `ams.color.brand.primary`?

Reply:
We decided to remove this token as it doesn’t reuse anything anyway – see Aram’s suggestion above.